index.html

```diff
@@ -94,7 +96,7 @@ <h2>Tonic</h2>
                             JavaScript
                         </li>
                     </ul>
-                    <button type="button" class="see-projects-btn btn">
+                    <button type="button" class="see-projects-btn one btn btn-1">
                         See projects
                     </button>
                 </div>
```

Comment:
- [ ] Your pop-up works great and looks like the given design. 👍 🌮 , But I think there are some requirements you need to meet before moving to the next milestone. Currently you still have the hard-coded project list in your index.html file, But according to the project requirement, both your project list and their details popup need to be generated dynamically. 👍 I'll suggest you remove the HTML code in the project work section and then dynamically display the list of projects on the home page from the array of project once the page loads. 😄 😄

> You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.